tests/compas/geometry/test_bbox.py

```diff
@@ -121,3 +123,37 @@ def test_oriented_bounding_box_numpy(coords, expected):
     results = oriented_bounding_box_numpy(coords)
     for result, expected_values in zip(results, expected):
         assert TOL.is_allclose(result, expected_values)
+
+
+@pytest.fixture
+def bbox_points_00():
+    return compas.json_load(os.path.join(HERE, "fixtures", "bbox_points_00.json"))
+
+@pytest.mark.parametrize(
+    "points, expected",
```

Comment:
i think that if you remove the space here, ipy will stop complaining...
`"points,expected"`

Reply:
ah ok... thank you will try this one

Reply:
hmmm still does not like it.. i guess ipy pytest does not know to use request.getfixturevalue